vk_video_decoder/libs/NvVideoParser/src/VulkanH264Parser.cpp

```diff
@@ -2255,7 +2255,7 @@ bool VulkanH264Decoder::slice_header(slice_header_s *slh, int nal_ref_idc, int n
     {
         if (slh->nhe.svc_extension_flag)
         {
-            slh->IdrPicFlag = m_nhe.svc.idr_flag;
+            slh->IdrPicFlag = (uint32_t)m_nhe.svc.idr_flag & 0x1;
```

Comment:
This should be:
slh->IdrPicFlag = (m_nhe.svc.idr_flag != 0);
Note this is different than:
m_nhe.svc.idr_flag & 0x1;